package.json

```diff
@@ -8,9 +8,10 @@
     "start:dev": "nodemon ./app.js",
     "start:test-server": "rm -f test.db && touch test.db && docker-compose down && docker-compose up -d --build test-server",
     "test:api": "CI=true newman run -n 3 -e tests/local.postman_environment.json tests/plusfries.postman_collection.json",
+    "build": "./scripts/widget.sh",
     "lint": "eslint .",
     "fmt": "prettier --no-color --write '**/*.{md,js}'",
-    "preinstall": "cp -n .env.example .env",
+    "preinstall": "test -f .env || cp -n .env.example .env",
```

Comment:
the `-n` flag should be `--no-clobber`. was this not the case on your end?

Reply:
without the check
```

Reply:
when i run `yarn install`

Reply:
Hm... 

```
       -n, --no-clobber
              do not overwrite an existing file (overrides a previous -i option)
```

there should be a flag like this. mind checking your `man cp`. maybe it's a little different? if not we can just do the `test`

Reply:
yep, it's there. might be because it writes to stdout?
```
❯ cp -n .env.example .env
.env not overwritten
```